### PR TITLE
Renaming "record" to "commit"

### PR DESCRIPTION
**Pull request by moh-27**
Hi,
this PR is for resolving issue #1070 by renaming the "record" to "commit"

I'm a beginner at this, let me know if I did something wrong so I can edit.
Thanks

### CODE REVIEW

File: packages/core/src/anchor/ethereum/ethereum-anchor-service.ts [hunk 1]
```diff
@@ -192,14 +192,14 @@ export default class EthereumAnchorService implements AnchorService {
           message: json.message,
         }
       case 'COMPLETED': {
-        const { anchorRecord } = json
```
Comment (ukstv) [inline packages/core/src/anchor/ethereum/ethereum-anchor-service.ts:195]: This change would make js-ceramic incompatible with [ceramic-anchor-service](https://github.com/ceramicnetwork/ceramic-anchor-service/blob/develop/src/models/request-presentation.ts#L30). I would advice adding a corresponding PR to ceramic-anchor-service repository to add `anchorCommit` field in addition to the present `anchorRecord` one. Please note: _add_, not _rename_. We want to maintain backwards compatibility.

**Review by ukstv — CHANGES_REQUESTED** [review]

**Comment by ukstv** [comment]
Apart from the single requested change related to interaction with ceramic-anchor-service, this thing is good. Good work @moh-27 !

**Comment by moh-27** [comment]
Thank you @ukstv for your review. And sorry for the late reply I got really busy.
@oed I'm gonna try doing it by today or tomorrow, and I let you know if I couldn't do it or didn't have time for it.

Thanks

**Comment by moh-27** [comment]
Hi @ukstv, I have done the requested changes and already have been [merged](https://github.com/ceramicnetwork/ceramic-anchor-service/pull/492) into the [ceramic-anchor-service](https://github.com/ceramicnetwork/ceramic-anchor-service/blob/develop/src/models/request-presentation.ts) repository.

**Comment by stbrody** [comment]
LGTM, merging.  Thanks!

**Pull request merged**